src/main/java/edu/oakland/translations/services/CoursesTranslationsService.java

```diff
@@ -1,24 +1,25 @@
-package edu.oakland.translations;
+package edu.oakland.translations.services;
 
-import edu.oakland.translations.LanguageStrings;
-
-import org.springframework.stereotype.Service;
+import org.springframework.stereotype.Component;
 import org.springframework.cache.annotation.Cacheable;
 
 import java.text.DateFormatSymbols;
 import java.text.SimpleDateFormat;
 import java.util.Locale;
 import java.util.Arrays;
 
-@Service
-public class LanguageStringsService implements ILanguageStringsService {
+import edu.oakland.translations.models.Translations;
+import edu.oakland.translations.models.CoursesTranslations;
+
+@Component("courses")
```

Comment:
Are you talking about the portlet fname? For courses the fname is `my-courses`. Using that name would be okay with me